docs/ci-cd-environment/machine-types.md

```diff
@@ -15,6 +15,10 @@ the supported machine types in your pipelines, refer to our [agent documentation
 
 Linux machine types can be paired with [Ubuntu 18.04 image][ubuntu1804], [Ubuntu 20.04 image][ubuntu2004], and [Docker-based environment][docker-env].
 
+### E1 Generation
+
+The `e1` series machines are our first generation, cost-effective with medium performance they are a good choice for less compute and memory intensive jobs. 
```

Comment:
I'd split the sentence into two to keep them shorter.

The `e1` series machines are our first generation, cost-effective with medium performance. They are a good choice for less compute and memory-intensive jobs.